coderd/prometheusmetrics/prometheusmetrics.go

```diff
@@ -516,6 +516,31 @@ func AgentStats(ctx context.Context, logger slog.Logger, registerer prometheus.R
 	}, nil
 }
 
+// Experiments registers a metric which indicates whether each experiment is enabled or not.
+func Experiments(_ slog.Logger, registerer prometheus.Registerer, exps []string, all codersdk.Experiments) error {
+	experimentsGauge := prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "coderd",
+		Name:      "experiments",
+		Help:      "Indicates whether each experiment is enabled (1) or not (0)",
```

Comment:
```suggestion
		Help:      "Indicates which experiments are available and whether an experiment is enabled (1) or not (0)",
```
Suggestion since I kind of read this as “either all experiments are enabled or they’re not”, so here’s an alternate proposal.

Reply:
Seems unnecessarily wordy to me, TBH; I think it's quite clear that an experiment which is not enabled is available.